app/src/main/java/edu/grinnell/grinnell_publications_android/Models/Interfaces/Story.java

```diff
@@ -0,0 +1,58 @@
+package edu.grinnell.grinnell_publications_android.Models.Interfaces;
+
+
+import java.util.List;
+
+/**
+ * @author Albert Owusu-Asare
+ * @since 1.1 Thu May  5 23:29:12 CDT 2016
+ * @see Publication
+ * @see Series
+ * @see Author
+ * @see Reactions
+ */
+public interface Story {
```

Comment:
In order to maintain consistency with our API, let's rename this to article. We don't to flip-flop around serializing "story" to "article" and vice-versa when we don't need do.